sdk/core/Azure.Core/src/Shared/DiagnosticScope.cs

```diff
@@ -1043,6 +1043,11 @@ public static void ResetFeatureSwitch()
     /// </summary>
     internal static class ActivityExtensions
     {
+        static ActivityExtensions()
```

Comment:
I think we also need to update the check in RequestActivityPolicy to respect the flag.

Reply:
Yep just added that as well